newdle/api.py

```diff
@@ -399,6 +400,9 @@ def send_result_emails(code):
     date = newdle.final_dt.strftime('%-d %B %Y')
     start_time = newdle.final_dt.strftime('%H:%M')
     end_time = (newdle.final_dt + newdle.duration).strftime('%H:%M')
+    event = create_calendar_event(newdle)
+    # Attachments have to be in the format: (filename, content_bytes, MIMEType)
```

Comment:
not sure if that comment is super useful here, it' kind of obvious that this is what the library expects..

if we want to document this, a better place would be the docstring of `notify_newdle_participants` which doesn't exist so for now i'd just not document it ;)

Reply:
Thanks, removed it.